### ISSUE TITLE
Add a "SignedInfo Conventions" section

### ISSUE THREAD

**Issue by ounsworth** [EntrustCorporation/draft-ounsworth-composite-sigs#136]
Make-like-picture from
https://datatracker.ietf.org/doc/html/draft-ietf-lamps-cms-sphincs-plus-03#name-signed-data-conventions

**Comment by ounsworth** [comment]
We also need to add SMIME-CAPS to the ASN.1. See the draft-cms-sphincs-plus

**Comment by johngray-dev** [comment]
Agreed.  Do we want to get this into -13, or wait until after adoption?

**Comment by johngray-dev** [comment]
Will address this after version 13 of the draft.

**Comment by johngray-dev** [comment]
Doing this saves writing a separate draft of using this draft in CMS...  Use link from Mike above SPHINCS template.
